ipam/allocator.go

```diff
@@ -45,24 +45,24 @@ type operation interface {
 // necessary plumbing.  Runs as a single-threaded Actor, so no locks
 // are used around data structures.
 type Allocator struct {
-	actionChan       chan<- func()
-	ourName          mesh.PeerName
-	universe         address.Range                // superset of all ranges
-	ring             *ring.Ring                   // information on ranges owned by all peers
-	space            space.Space                  // more detail on ranges owned by us
-	owned            map[string][]address.Address // who owns what addresses, indexed by container-ID
-	nicknames        map[mesh.PeerName]string     // so we can map nicknames for rmpeer
-	pendingAllocates []operation                  // held until we get some free space
-	pendingClaims    []operation                  // held until we know who owns the space
-	dead             map[string]time.Time         // containers we heard were dead, and when
-	db               db.DB                        // persistence
-	gossip           mesh.Gossip                  // our link to the outside world for sending messages
-	paxos            *paxos.Node
-	paxosActive      bool
-	ticker           *time.Ticker
-	shuttingDown     bool // to avoid doing any requests while trying to shut down
-	isKnownPeer      func(mesh.PeerName) bool
-	now              func() time.Time
+	actionChan        chan<- func()
+	ourName           mesh.PeerName
+	universe          address.Range                // superset of all ranges
+	ring              *ring.Ring                   // information on ranges owned by all peers
+	space             space.Space                  // more detail on ranges owned by us
+	owned             map[string][]address.Address // who owns what addresses, indexed by container-ID
+	nicknames         map[mesh.PeerName]string     // so we can map nicknames for rmpeer
+	pendingAllocates  []operation                  // held until we get some free space
+	pendingClaims     []operation                  // held until we know who owns the space
+	dead              map[string]time.Time         // containers we heard were dead, and when
+	db                db.DB                        // persistence
+	gossip            mesh.Gossip                  // our link to the outside world for sending messages
+	participant       paxos.Participant
```

Comment:
To my mind, the variable named `participant` could have been left as `paxos`; "participant" doesn't seem to add anything at the point of use to indicate it may be a real Paxos participant or an observer.